tests/conftest.py

```diff
@@ -22,10 +22,13 @@
 from server.lobbyconnection import LobbyConnection
 from server.matchmaker import MatchmakerQueue
 from server.player_service import PlayerService
+from server.rating_service.rating_service import RatingService
 from server.players import Player, PlayerState
 from server.rating import RatingType
 from tests.utils import MockDatabase
 
+pytestmark = pytest.mark.asyncio
```

Comment:
Conftest doesn't have any tests in it so you shouldn't need this line

Reply:
True! Took me some time to figure out how to make everything use the same event loop, at some point I thought this was necessary...